packages/react-select/src/Select.js

```diff
@@ -1917,14 +1788,32 @@ export default class Select extends Component<Props, State> {
   }
 
   renderLiveRegion() {
-    if (!this.state.isFocused) return null;
+    const { commonProps } = this;
+    const {
+      ariaSelection,
+      focusedOption,
+      focusedValue,
+      isFocused,
+      selectValue,
+    } = this.state;
+
+    const { inputValue, menuIsOpen, options } = this.props;
```

Comment:
Should we just have `LiveRegion` pull these off of `selectProps` inside the `LiveRegion` component like it does for a lot of the other props?